dist/api_data.py

```diff
@@ -1353,6 +1353,11 @@ def __ge__(self, other):
         Display the contents of on-disk blocks as they are verified,
         using the application's message handler, intended for debugging''',
         type='boolean'),
+    Config('dump_history', 'false', r'''
+        Display values and start and stop time pairs as keys are verified
```

Comment:
Display all values for a key along with their time pairs as keys are verified against the history store. Might be a better phrasing?

Reply:
How's 
"Display a key's values along with its start and stop time pairs as
        they are verified against the history store,..."

Reply:
Sounds good